pyls/plugins/jedi_completion.py

```diff
@@ -173,7 +176,7 @@ def _label(definition):
 def _detail(definition):
     try:
         return definition.parent().full_name or ''
-    except AttributeError:
+    except (AttributeError, TypeError):
```

Comment:
Why is TypeError necessary now and was not before? `_detail` is always called with `definition` being a Completion, which always implements parent().

Reply:
You're right. This is a remnant from cb51b09254efac0b2c22130815198fab487db54f when `definition` was not a completion.